commands/show/nodepool/command.go

```diff
@@ -136,12 +136,15 @@ func fetchNodePool(args arguments) (*result, error) {
 
 	res.instanceTypeDetails, err = awsInfo.GetInstanceTypeDetails(res.nodePool.NodeSpec.Aws.InstanceType)
 	if err != nil {
-		return nil, microerror.Mask(err)
+		// We deliberately ignore "instance type not found", but respect all other errors.
+		if !nodespec.IsInstanceTypeNotFoundErr(err) {
```

Comment:
Let's please stick to https://github.com/giantswarm/fmt/blob/master/go/errors.md#matching-errors and have it without indentation. It is way easier to understand that way and the greater level of consistency makes it very easy to work with arbitrary projects.

Reply:
Done